1_processing/README.md

```diff
@@ -1,29 +1,31 @@
-In our exploratory analysis we found that samples labeled as PAO1 based on SRA annotations had high PAO1 accessory gene expression.
-Whereas samples labeled as PA14 by SRA had high PA14 accessory gene expression.
+To determine which strains are PAO1 versus PA14 we will use the median expression of accessory genes to determine if a strain is PAO1 or PA14.
+In our exploratory analysis we found that strains labeled as PAO1 based on SRA annotations had high PAO1 accessory gene expression.
+Whereas strains labeled as PA14 by SRA had high PA14 accessory gene expression.
 
 See plot below where the median expression of PAO1 genes (PAO1 accessory genes) on the x-axis and the median expression of PA14-only genes (PA14 accessory genes) on the y-axis.
-Each point is a sample.
+Each point is a strain/sample.
 ![all_samples](https://github.com/greenelab/core-accessory-interactome/blob/master/0_explore_data/Expression_accessory_genes_all_samples.svg)
 
-A sample is considered PAO1 if the median gene expression of PA14 accessory genes is 0 and PAO1 accessory genes in > 5.
-Similarlty, a sample is considered PA14 if the median gene expression of PA14 accessory genes is > 5 and PAO1 accessory genes in 0.
+A strain is considered PAO1 if the median gene expression of PA14 accessory genes is 0 and PAO1 accessory genes in > 5.
+Similarlty, a strain is considered PA14 if the median gene expression of PA14 accessory genes is > 5 and PAO1 accessory genes in 0.
 
-A threshold of 25 MR normalized estimated counts is used based on our analysis in [1_decide_threshold.ipynb](1_decide_threshold.ipynb). The goal of this notebook was to  define a threshold to determine if a sample if PAO1 or not (likewise, if a sample is PA14 or not). We used known labels from SRA to do this. Specifically, we examined the distribution of PAO1 samples (grey) vs non-PAO1 samples (blue). We define the threshold to be one that separated between the two distributions. We use this threshold in [2_create_compendia.ipynb](2_create_compendia.ipynb) to partition gene expression data into PAO1 and PA14 compendia.because we found that using a threshold of 0 MR normalized estimated counts included some other SRA-labeled strains.
+A threshold of 25 MR normalized estimated counts is used based on our analysis in [1_decide_threshold.ipynb](1_decide_threshold.ipynb). The goal of this notebook was to  define a threshold to determine if a strain if PAO1 or not (likewise, if a strain is PA14 or not). We used known labels from SRA to do this. Specifically, we examined the distribution of PAO1 strain vs non-PAO1 strain (see histogram plots). We define the threshold to be one that separated between the two distributions. We use this threshold in [2_create_compendia.ipynb](2_create_compendia.ipynb) to partition gene expression data into PAO1 and PA14 compendia.because we found that using a threshold of 0 MR normalized estimated counts included some other SRA-labeled strains.
```

Comment:
```suggestion
A threshold of 25 MR normalized estimated counts is used based on our analysis in [1_decide_threshold.ipynb](1_decide_threshold.ipynb). The goal of this notebook was to  define a threshold to determine if a strain if PAO1 or not (likewise, if a strain is PA14 or not). We used known labels from SRA to do this. Specifically, we examined the distribution of PAO1 strain vs non-PAO1 strain (see histogram plots). We define the threshold to be one that separated between the two distributions. We use this threshold in [2_create_compendia.ipynb](2_create_compendia.ipynb) to partition gene expression data into PAO1 and PA14 compendia because we found that using a threshold of 0 MR normalized estimated counts included some other SRA-labeled strains.
```